translations/strings.yaml

```diff
@@ -194,6 +194,24 @@ genericPurchaseError:
   genericPurchaseErrorText: An unexpected error has occurred, please try again. Visit our help center below to learn how to troubleshoot this issue or contact support.
   genericPurchaseErrorButton: Try Again
 
+# IAP iOS restore purchase button
+restorePurchase:
+  restorePurchaseButton: Already a subscriber?
+
+# IAP iOS restore expired error
+restorePurchaseExpiredError:
+  restorePurchaseExpiredErrorText:  Sorry we are unable to connect your Firefox Account to a current subscription. Please try again or contact our support team for further assistance.
+
+# IAP iOS restore in use error
+restorePurchaseInUseError:
+  restorePurchaseInUseErrorHeader: Error confirming subscription
+  restorePurchaseInUseErrorText: Another Firefox Account has already subscribed using this Apple ID. Visit our help center below to learn more about how to manage your subscriptions.
```

Comment:
Do we want to keep the `help center below` part in `restorePurchaseInUseErrorText` and `restorePurchaseNotFoundErrorText`? See discussion here: https://github.com/mozilla-mobile/mozilla-vpn-client/pull/2414/files/8f959b0efc4192dbe2919a75bf7c07f22c2ac83e#r772131120.